homework-2.0/task3.rb

```diff
@@ -0,0 +1,19 @@
+class ArrayFilter
+  def initialize(arr)
+    @arr = arr
+  end
+
+  def filter
+    @arr.select { |item| (to_numeric(item) % 2).zero? }
+  end
+
+  private
+
+  def to_numeric(thing)
+    return thing.to_s.to_i if thing.to_s == thing.to_s.to_i.to_s
```

Comment:
`to_s.to_i` doesn't look too good. Mayby consider other solution?

Please let me know if you need any help

Reply:
As an option you can use `regex`

Reply:
Is my new solution satisfactory or can it be done more correctly?